client/my-sites/email/inbox/mailbox-selection-list/index.jsx

```diff
@@ -138,27 +141,62 @@ const MailboxListStatus = ( { isError, statusMessage } ) => {
 
 MailboxListStatus.propType = {
 	isError: PropTypes.bool,
-	statusMessage: PropTypes.string.isRequired,
+	statusMessage: PropTypes.node.isRequired,
+};
+
+const MailboxLoaderError = ( { refetchMailboxes, siteId } ) => {
+	const translate = useTranslate();
+	const queryClient = useQueryClient();
+
+	const reloadMailboxes = () => {
+		queryClient.removeQueries( getMailboxesCacheKey( siteId ) );
+		refetchMailboxes();
+	};
+
+	return (
+		<div className="mailbox-selection-list__loader-error-container">
+			<section>
+				<img src={ unsupportedBrowserIllustration } alt={ translate( 'Google Workspace icon' ) } />
```

Comment:
We should change the value of that `alt` attribute.

Reply:
Wow. Thanks!